beacon-chain/operations/attestations/kv/kv.go

```diff
@@ -4,8 +4,11 @@ import (
 	"sync"
 
 	ethpb "github.com/prysmaticlabs/ethereumapis/eth/v1alpha1"
+	"github.com/prysmaticlabs/prysm/shared/hashutil"
 )
 
+var hashFn = hashutil.HashProto
```

Comment:
Did you make this global to make changing the hash type easier?

Reply:
yes. i wanted to make it easier to change again in the future. Maybe we would change this to a ssz based hashing which can be even faster with the generated marshal and unmarshal functions